pkg/external_tools/mcp/mcp.go

```diff
@@ -0,0 +1,60 @@
+package external_tools
+
+import (
+	"github.com/modelcontextprotocol/go-sdk/mcp"
+	kubernetesmcp "github.com/ovn-kubernetes/ovn-kubernetes-mcp/pkg/kubernetes/mcp"
+)
+
+// MCPServer provides MCP server functionality for external tools operations.
+type MCPServer struct {
+	k8sMcpServer *kubernetesmcp.MCPServer
+}
+
+// NewMCPServer creates a new MCP server instance
+func NewMCPServer(k8sMcpServer *kubernetesmcp.MCPServer) *MCPServer {
+	return &MCPServer{
+		k8sMcpServer: k8sMcpServer,
+	}
+}
+
+// AddTools registers external tools with the MCP server
+func (s *MCPServer) AddTools(server *mcp.Server) {
+	mcp.AddTool(server,
+		&mcp.Tool{
+			Name: "tcpdump",
+			Description: `Capture network packets with strict safety controls.
+
+Parameters:
+- packet_count: Number of packets to capture (default: 1000, max: 1000)
```

Comment:
I think default can be set to 100 as 1000 by default may be a little too high value.